components/widgets/admin_console/admin_header.tsx

```diff
@@ -5,13 +5,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default class AdminHeader extends React.Component {
-    static get propTypes() {
+    public static get propTypes() {
         return {
             children: PropTypes.node.isRequired,
         };
     }
 
-    render() {
+    public render(): React.ReactNode {
```

Comment:
I don't think return type is required. It's inferred by TS

Reply:
Is not, is required by the linter.